daemon/containerd/image_inspect.go

```diff
@@ -97,7 +98,10 @@ func (i *ImageService) ImageInspect(ctx context.Context, refOrID string, opts im
 				LastTagTime: lastUpdated,
 			},
 		},
-		Parent: parent, // field is deprecated with the legacy builder, but returned by the API if present.
+		// GraphDriver is omitted in API v1.52 unless using a graphdriver.
+		GraphDriver: &storage.DriverData{Name: i.snapshotter},
```

Comment:
I'll have a quick look on my machine, but I suspect we may not need the field here, as with snapshotters, the name of the driver is available in the response, only in a different field, so we could handle it in the router based on API version (on API < v1.52 copy the field to the GraphDriver struct);
https://github.com/moby/moby/blob/79912d4c7f40c9432a83157e6834098d60729351/daemon/inspect.go#L157-L172

Reply:
Yeah, I just noticed indeed; I pushed a commit with a slight alternative, but not 100% which one is cleaner 😂 - pushed it to get your input / thoughts.